### PR TITLE
[RHMAP 5595] Modifications to support for the new model definitions for mbaas targets

### PR DESCRIPTION
**Pull request by paolobueno**
Tied to the changes in fh-supercore:env-mbaas-dev, thus PRing against a feature branch of the same name.
Please check a3d2b08 for specific changes

ping @pb82 @wtrocki

### CODE REVIEW

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* There should be an associated fhcap PR with this change - please provide a link to this
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.

In order to update fhcap with your component change please execute:
`fhcap component update fh-fhc 2.6.0 250`

**Comment by paolobueno** [comment]
ping @rsmeral

File: lib/cmd/fh3/admin/mbaas/.delete.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// 2016-04-07: This command was temporarily disabled due to decision of not supporting mbaas deletion
```
Comment (pb82) [inline lib/cmd/fh3/admin/mbaas/.delete.js]: Why is this file hidden? It's called  `.delete.js`

Comment (paolobueno) [inline lib/cmd/fh3/admin/mbaas/.delete.js]: I thought it was a nice way to "disable" the command, do you have some recommendation for a better way?

**Comment by pb82** [comment]
LGTM

**Comment by rsmeral** [comment]
@paolobueno I agree on the capitalization of "MBaaS". It's been established some terms here: https://docs.google.com/document/d/1QiqIRDcujfNxDgABjL0OhSPa--b2I7XHzk-GF-_xyCs/edit#
The capitalization is not yet fully clarified there, but MBaaS was agreed on.

**Comment by paolobueno** [comment]
Thanks for the reviews

@pb82 I'll name the file _delete.js, this way we won't have problems with unix hiding it from anyone, and it probably won't be affected by future changes to `.gitignore`

@rsmeral Thanks, I see no reason to revert this change, now that we're doing some work on this part of FHC. A quick googling for mbaas also shows that "MBaaS" is the much more common form.
There's probably some lone variables with varying capitalization, but I think I got all the user-facing strings.

Merging to the feature branch, no version bump yet...

**Comment by rsmeral** [comment]
@paolobueno Yeah, that's what I meant (though I wasn't clear): Let's change it to MBaaS here (as you've done), since that's prevalent. Thanks.

**Pull request merged**